src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -793,4 +802,46 @@ public void itemRecalledThenCancelledAndNextRecallDoesNotChangeDueDate() {
     assertThat("due date after recall should not change the renewal due date",
         recalledRenewalDueDate, is(renewalDueDate));
   }
+
+  @Test
+  public void shouldNotRenewOverdueLoan() {
```

Comment:
This test does not seem to attempt to renew the loan. Have I missed something?

Reply:
Renamed test method to `shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled`

Reply:
Ah, thank you. I realised, once I got chance to read the JIRA issue that I had misunderstood the issue